docs/articles/Core/ResourceReference.md

```diff
@@ -0,0 +1,25 @@
+# ResourceReference
+
+`ResourceReference` is an attribute that links to resources together. 
```

Comment:
maybe `several` resources, since it can also be used for a list (see channels in IMessageDriver)